### ISSUE TITLE
`BacksolveAdjoint` does not work with `FixedStepController`

### ISSUE THREAD

**Issue by DesmondZhong** [martenlienen/torchode#28]
Hello,

Thank you for the nice library which overcome the limitations of torchdiffeq and torchdyn. 

During my exploration with the library, I found that when I use `BacksolveAdjoint`, I cannot use `FixedStepController`. Here's a minimum example modified from the documentation to reproduce the issue.

```python
import torch
import torchode as to

class Model(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.linear = torch.nn.Linear(1, 1)

    def forward(self, t, y):
        return self.linear(y)
    

y0 = torch.tensor([[1.2], [5.0]])
n_steps = 10
t_eval = torch.stack((torch.linspace(0, 5, n_steps), torch.linspace(3, 4, n_steps)))

model = Model()
term = to.ODETerm(model)
step_method = to.Dopri5(term=term)

step_size_controller = to.FixedStepController()
solver = to.BacksolveAdjoint(term, step_method, step_size_controller)

sol = solver.solve(
    to.InitialValueProblem(y0=y0, t_eval=t_eval),
    dt0=t_eval[:, 1] - t_eval[:, 0]
)
```

I got the following error
```
File "...\site-packages\torchode\step_size_controllers.py", line 139, in init
    assert dt0 is not None, "Fixed step size solving requires you to configure dt0"
           ^^^^^^^^^^^^^^^
AssertionError: Fixed step size solving requires you to configure dt0
```

In `BacksolveFunction`, it seems `solver.solve` is called by explicitly setting `dt0=None` which raises this assertion error when using `FixedStepController`. Is this intended or is it a bug?

Thank you very much!

**Comment by martenlienen** [comment]
Thanks for reporting this. I have fixed it in a new version 0.1.9. You can now pass `dt0` to `solver.solve` and the backward solve will use the same timestep by default. You can also call `solver.solve(..., dt0=dt0, backward_dt0=-5 * dt0)` or similar if you want a separate step for the backward pass. Note that the backward timestep needs to be of opposite sign as the forward timestep.

Update with `pip install -U torchode`.

**Comment by DesmondZhong** [comment]
Thank you Marten for resolving the issue so quickly!
